js/app/Data/CartoDBInfoWindow.js

```diff
@@ -78,7 +77,9 @@ define([
          obj
         );
 
-        cb($(popupHtml).find('.cartodb-popup-content'));
+        var html = $(popupHtml).find('.cartodb-popup-content');
+
+        cb(html, data);
```

Comment:
This is confusing cb with two args is usually cb(err, data).
Is there a big reason you didn't do 

    obj.html = html;
    obj.toString = function () { return this.html);
    return obj;

? This is a pattern I've used in many places...